governance/polls/Inclusion Poll - Core Unit MIP Set- March 8, 2021.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Inclusion Poll for Core Unit MIP Set (MIP38-41 and Subproposals) - March 8, 2021
+summary: Signal your support or opposition for this proposal being included in this month's governance cycle. 
+discussion_link: https://forum.makerdao.com/t/road-to-dao-the-core-unit-operating-model/6105
+vote_type: Plurality Voting
+categories:
+   - Governance
+   - MIPs
+   - Core Unit
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Inclusion Poll for Core Unit MIP Set (MIP38-41 and Subproposals) - March 8, 2021
+
+The Governance Facilitators have placed an inclusion poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP3](https://github.com/makerdao/mips/blob/master/MIP3/mip3.md). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March 8 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to this proposal being included in this month's governance cycle.
+
+ This Inclusion Poll MIP Set contains the following MIPs:
+* [MIP38: DAO Primitives State](https://forum.makerdao.com/t/mip38-dao-primitives-state/6095)
+* [MIP39: Core Unit Framework](https://forum.makerdao.com/t/mip39-core-unit-framework/6096)
+* [MIP40: Budget Framework](https://forum.makerdao.com/t/mip40-budget-framework/6097)
+* [MIP41: Facilitator Framework](https://forum.makerdao.com/t/mip41-facilitator-framework/6098)
+* [MIP4c2-SP10: MIP0 Amendments ](https://forum.makerdao.com/t/mip4c2-sp10-mip0-amendments/6099)
+* [MIP4c2-SP12: MIP16 Amendments](https://forum.makerdao.com/t/mip4c2-sp12-mip16-amendments/6101)
+
+Please review the discussion [thread](https://forum.makerdao.com/t/road-to-dao-the-core-unit-operating-model/6105) to inform your position on this proposal before voting. It was also mentioned and possibly discussed on the [Governance and Risk call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-131-thursday-march-4-17-00-utc/6737) on March 4, 2021.
+
+## Outcomes
+
+This poll implements an **Inclusion Threshold.** The Inclusion Threshold is currently set to **3,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option by at least 3,000 MKR then the following actions will be taken:**
+* The Governance Facilitator(s) will confirm the inclusion of this proposal within the next Monthly Governance Poll on the Governance and Risk call on March 11, 2021. 
```

Comment:
this proposal -> these proposals

Reply:
We should maybe have started by creating a new template for a MIP Sets inclusion poll. We should do that for next month.